package.json

```diff
@@ -51,6 +51,7 @@
     "loveboat": ">=1.5.0 <2",
     "renamer": "0.5.x",
     "seneca-echo": "0.3.x",
+    "sqlite3": "3.1.x",
```

Comment:
Schwifty, yes (pending publishing) but I _think_ objection/knex will come in fine with schwifty.